migrations/Version202607281623000000_taoEventLog.php

```
@@ -0,0 +1,44 @@
+use oat\taoItems\model\event\ItemContentViewEvent;
+use oat\taoItems\model\event\ItemPrintAttemptEvent;
+use oat\taoTests\models\event\TestContentViewEvent;
+
+final class Version202607281623000000_taoEventLog extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Attach preview events to LoggerService';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)